dashboard/app/controllers/survey_results_controller.rb

```diff
@@ -0,0 +1,7 @@
+class SurveyResultsController < ApplicationController
+
+  def create
+    # puts params[:survey].to_json
```

Comment:
How do I get these parameters into the serialized_attr properties?  Do I need to create a serializer?  Is there an automated way to do that?